hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ECContainerReplicaCount.java

```diff
@@ -127,6 +128,9 @@ public ECContainerReplicaCount(ContainerInfo containerInfo,
       3 is not considered over replicated because its second copy is unhealthy.
       */
       if (replica.getState() == ContainerReplicaProto.State.UNHEALTHY) {
+        int val = decommissionIndexes
+            .getOrDefault(replica.getReplicaIndex(), 0);
+        unHealthyIndexes.put(replica.getReplicaIndex(), val + 1);
```

Comment:
Don't understand this part. Did you mean to get from the `unHealthyIndexes` map?

Reply:
Ah well spotted. This is a copy and paste error. `decommissionIndexes` should be `unHealthyIndexes` here. I will fix this.